client/components/site-selector/index.jsx

```diff
@@ -311,6 +313,23 @@ export default React.createClass( {
 						{ this.renderAllSites() }
 						{ this.renderStarredSites() }
 						{ this.renderSites() }
+						{ hiddenSitesCount > 0 &&
+							<span className="site-selector__hidden-sites-message">
+								{ this.translate(
+									'%(hiddenSitesCount)d more hidden site.{{br/}} Use search to access it.',
```

Comment:
Doesn't appear to affect the design from the screenshot, but there's an unnecessary space at the start of the second line of this text.